apps/storybook-react/tailwind.config.js

```diff
@@ -9,4 +9,9 @@ module.exports = {
     './src/**/*.{js,jsx,ts,tsx}',
     './stories/**/*.{js,jsx,ts,tsx}',
   ],
+  theme: {
+    colors: {}, // This removes all default Tailwind colors. We want to rely on the colors provided by @metamask-previews/design-system-tailwind-preset
+    extend: {},
+  },
+  plugins: [],
```

Comment:
We don't want to be able to use the Tailwind default colors like bg-black bg-blue500 etc

Reply:
Moved to a separate PR https://github.com/MetaMask/metamask-design-system/pull/683